dev-server/ui-playground/MenuDemo.js

```diff
@@ -0,0 +1,29 @@
+import {
+  PatternPage,
+  Pattern,
+  PatternExamples,
+  PatternExample,
+} from './Patterns';
+
+import Menu from '../../src/sidebar/components/Menu';
+import MenuItem from '../../src/sidebar/components/MenuItem';
+
+export default function ButtonsSharedDemo() {
+  return (
+    <PatternPage title="Menu">
+      <Pattern title="Menu">
+        <p>A simple Menu usage example</p>
+        <PatternExamples>
```

Comment:
`title` is not being used, do you anticipate other patterns to use it?

Reply:
Yep! I'm using it in the Buttons demos that haven't been added yet.